index.js

```diff
@@ -1,95 +1,122 @@
 
-// console.log(name)
-// // => "Ian"
-// name = "Not Ian" 
-// // => TypeError
-// console.log("%c----------", "color: red") 
+const name = 'SunJet';
 
+console.log("%cQuestion 2", "color: red") 
 
+console.log(name)
+// => "Ian"
+//name = "Not Ian" 
+// => TypeError
+console.log("%c----------", "color: red") 
 
 // ***** Question 3 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
-
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
-
+function drinkWater(currentThirstLevel) {
+    console.log(`"Man I sure am thirsty"`)
+    currentThirstLevel -= 1
+    console.log(`"Ahh that hits the spot"`)
+    return currentThirstLevel
+}
+    
+console.log("%cQuestion 3", "color: red") 
 
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 // ***** Question 4 *****
+function sameSameButDifferent(num,maybeNum){
+  if (num === maybeNum) {
+    return '"same same"';
+  }
+  else if (num === parseInt(maybeNum)) {
+    return '"same same (but different)"';
+  } 
+  else {
+    return '"different"';
+  }
+}
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
 
-// console.log(sameSameButDifferent(5, 5)) 
-// // => "same same"
-//
-// console.log(sameSameButDifferent(123, "123")) 
-// // => "same same (but different)"
-//
-// console.log(sameSameButDifferent(5, 7)) 
-// // => "different"
-//
-// console.log(sameSameButDifferent(123, "122")) 
-// // => "different"
-// console.log("%c----------", "color: red") 
+console.log(sameSameButDifferent(5, 5)) 
+// => "same same"
 
+console.log(sameSameButDifferent(123, "123")) 
+// => "same same (but different)"
 
+console.log(sameSameButDifferent(5, 7)) 
+// => "different"
 
-// ***** Question 5 *****
+console.log(sameSameButDifferent(123, "122")) 
+// => "different"
+console.log("%c----------", "color: red") 
 
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
+// ***** Question 5 *****
 
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)  
-// // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red") 
+function updateGrade(student,grade){
+  student.grade = grade
+}
 
+// *** Uncomment the lines below to test
+console.log("%cQuestion 5", "color: red")
 
+const student1 = { name: "Duane", grade: 88 }
+updateGrade(student1, 92)
+console.log(student1)  
+// => { name: "Duane", grade: 92 }
+console.log("%c----------", "color: red") 
 
 // ***** Question 6 *****
 
+function printNameAndPhones(users){
+  users.forEach (user => {
+    console.log(`"${user.name}"`);
+    console.log(`"Cell: ${user.phones.cell}"` );
+    console.log(`"Ofice: ${user.phones.office} "`);
+  });
```

Comment:
Arrow function! Love to see it. There are a few tricky things about arrow functions but I'll have a video to go over them later in the week.